src/test/java/tech/jhipster/lite/module/infrastructure/secondary/javadependency/gradle/GradleCommandHandlerTest.java

```diff
@@ -497,15 +497,15 @@ void shouldNotDuplicateExistingProfileWithDifferentActivation() {
   @Nested
   class HandleAddDirectJavaDependency {
 
-    private final JHipsterProjectFolder projectFolder = projectFrom("src/test/resources/projects/empty-gradle");
+    private final JHipsterProjectFolder jhProjectFolder = projectFrom("src/test/resources/projects/empty-gradle");
```

Comment:
I don't agree with this rule, I don't think it is relevant un Java

Reply:
It don't have any issue with this rule, I think it highlights some naming clumsiness, but I don't agree with the new name which is not expressive enough.
```suggestion
    private final JHipsterProjectFolder emptyGradleProjectFolder = projectFrom("src/test/resources/projects/empty-gradle");
```